src/conversion.jl

```diff
@@ -92,39 +98,48 @@ end
 for ((TA, fieldA), (TB, fieldB)) in (((MatrixMap, :lmap), (MatrixMap, :lmap)),
                                      ((MatrixMap, :lmap), (UniformScalingMap, :λ)),
                                      ((UniformScalingMap, :λ), (MatrixMap, :lmap)))
-    @eval function Base.convert(::Type{AbstractMatrix}, AB::CompositeMap{<:Any,<:Tuple{$TB,$TA}})
+    @eval function Base.convert(::Type{AbstractMatrix},
+                                AB::CompositeMap{<:Any,<:Tuple{$TB,$TA}})
         B, A = AB.maps
         return A.$fieldA*B.$fieldB
     end
 end
-function Base.Matrix{T}(AB::CompositeMap{<:Any,<:Tuple{MatrixMap,MatrixMap}}) where {T}
+function Base.Matrix{T}(AB::CompositeMap{<:Any, <:Tuple{MatrixMap, MatrixMap}}) where {T}
     B, A = AB.maps
     return convert(Matrix{T}, A.lmap*B.lmap)
 end
-function SparseArrays.sparse(AB::CompositeMap{<:Any,<:Tuple{MatrixMap,MatrixMap}})
+function SparseArrays.sparse(AB::CompositeMap{<:Any, <:Tuple{MatrixMap, MatrixMap}})
     B, A = AB.maps
     return convert(SparseMatrixCSC, A.lmap*B.lmap)
 end
-function Base.Matrix{T}(λA::CompositeMap{<:Any,<:Tuple{MatrixMap,UniformScalingMap}}) where {T}
+function Base.Matrix{T}(λA::CompositeMap{<:Any, <:Tuple{MatrixMap, UniformScalingMap}}) where {T}
     A, J = λA.maps
     return convert(Matrix{T}, J.λ*A.lmap)
+    # if you want less allocations: lmul!(J.λ, convert(Matrix{T}, A.lmap))
```

Comment:
I just tested this: it would change the underlying `A.lmap`, which is probably not what we want.

Reply:
Ok I did not consider that.